packages/api/src/jobs/projects-cleanup.ts

```diff
@@ -0,0 +1,35 @@
+import sql from "sql-template-strings";
+import { initClients } from "../app-router";
+import { CliArgs } from "../parse-cli";
+import { DB } from "../store/db";
+import Queue from "../store/queue";
+import { triggerCleanUpProjectsJob } from "../controllers/project";
+import { Request } from "express";
+
+// queries for all the deleted projects
+// clean up logic for all related assets and streams
+export default async function projectsCleanup(
+  config: CliArgs,
+  req: Request,
+  clients?: { jobsDb: DB },
+) {
+  if (!config.ingest?.length) {
+    throw new Error("ingest not configured");
+  }
+  const { jobsDb } =
+    clients ?? (await initClients(config, "projects-cleanup-job"));
+  const { projectsCleanupLimit: limit, ingest } = config;
+
+  let [projects] = await jobsDb.stream.find([sql`data->>'deleted' = 'true'`], {
+    limit,
+    order: "data->>'lastSeen' DESC",
+  });
```

Comment:
We probably need sth like a "`deleting` phase" on the projects too, otherwise this will keep listing deleted files that have already been cleaned-up ad infinitum. Maybe a simple `cleanedUp` boolean in the project that gets set by this job?